### PR TITLE
github-cli: update to 1.0.0.

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

**Comment by NeelChotai** [comment]
1.0.0 is out.

File: srcpkgs/github-cli/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'github-cli'
 pkgname=github-cli
-version=0.11.1
+version=0.12.0
```
Comment (mcbloch) [inline srcpkgs/github-cli/template:3]:
```suggestion
version=1.0.0
```

File: srcpkgs/github-cli/template [hunk 2]
```diff
@@ -12,7 +12,7 @@ license="MIT"
 homepage="https://cli.github.com"
 changelog="https://github.com/cli/cli/releases"
 distfiles="https://github.com/cli/cli/archive/v${version}.tar.gz"
-checksum=5ec8bab6a2c6f26cabc2a6b5f301d8971f9f654a85c6da5e7203e5e2d94cbec0
+checksum=227031cc381555da2a406bde7d5bfdd631979f635cda22537f206b5362cf6fc1
```
Comment (mcbloch) [inline srcpkgs/github-cli/template:15]:
```suggestion
checksum=96575c888d979a6717928fe53bbd75d1acd52a6eec32aa41202d15919475ebfa
```

**Review by mcbloch — CHANGES_REQUESTED** [review]
I'd suggest bumping to 1.0

**Comment by shizonic** [comment]
Updated to 1.0.0. Thanks!

**Pull request merged**